src/pages/waiting-room-to-car/waiting-room-to-car.html

```diff
@@ -9,106 +9,200 @@
+                  Select the eyesight test outcome
+                  </div>
+          </ion-row>
+        </ion-col>
+      </ion-row>
         <ion-row *ngIf="pageState.eyesightFailRadioChecked$ | async">
           <eyesight-failure-confirmation [cancelFn]="eyesightFailCancelled"></eyesight-failure-confirmation>
         </ion-row>
-        <ion-row [hidden]="pageState.eyesightFailRadioChecked$ | async" id="post-eyesight-form-content">
-          <ion-col>
-            <ion-row class="mes-full-width-card" id="tell-me-card" align-items-center>
-              <ion-col col-32>
+        <ion-row no-padding [hidden]="pageState.eyesightFailRadioChecked$ | async" id="post-eyesight-form-content">
+          <ion-col class="no-padded-col">
+            <ion-row class="mes-validated-row" id="tell-me-card" align-items-center justify-content-start>
+              <div class="row-validation-bar"></div>
+              <ion-col class="component-label" col-32>
                 <label>Tell me question</label>
               </ion-col>
-            </ion-row>
-            <ion-row class="mes-full-width-card" id="registration-card" align-items-center>
-              <ion-col col-32>
-                <label>Registration number</label>
+              <ion-col>
+                  <ion-row class="spacing-row"></ion-row>
+                  <ion-row class="component-row" align-items-centre justify-content-start >
+                      <ion-col></ion-col>
+                      <ion-col></ion-col>
+                      <ion-col></ion-col>
+                  </ion-row>
+                  <ion-row class="validation-message-row" align-items-centre justify-content-start>
+                  </ion-row>
+              </ion-col>  
+              </ion-row>
+
+            <ion-row class="mes-validated-row" id="registration-card" align-items-center>
+                <div class="row-validation-bar" [class.ng-invalid]="isCtrlDirtyAndInvalid('registrationNumberCtrl')"></div>
+                <ion-col class="component-label" col-32>                
+                  <label>Vehicle registration number</label>
               </ion-col>
               <ion-col>
-                <input #registrationInput type="text" id="registration" autocapitalize="characters" uppercaseAlphanumOnly>
+                <ion-row class="spacing-row"></ion-row>
+                <ion-row class="component-row" align-items-centre justify-content-start >
+                    <ion-col col-32>
+                        <input #registrationInput type="text" id="registration" formControlName="registrationNumberCtrl"
+                        class="vehicle-reg-input" autocapitalize="characters" uppercaseAlphanumOnly
+                        [class.ng-invalid]="isCtrlDirtyAndInvalid('registrationNumberCtrl')"
+                        [value]="pageState.registrationNumber$ | async">
+                      </ion-col>
+                    <ion-col></ion-col>
```

Comment:
ditto above comment

Reply:
It's more so people are aware that in the design it was envisaged there would be 3 columns. in the component section, but I'm happy to remove them if you feel they add no value.

Reply:
After consideration have removed the empty ion-cols